config/database.yml.example

```diff
@@ -1,25 +1,21 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
+  adapter: mysql2
+  encoding: utf8
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
+  username: your'username
+  password: your'pass
+  socket: /var/run/mysqld/mysqld.sock
```

Comment:
you should use gem dotenv (https://github.com/bkeepers/dotenv) to write security variables such as username, password in env file. Otherwise others can see them

Reply:
isnt the dotenv applied when the password is exposed? In this example file, there is no exposed username and password. Should i still nic to applied dotenv?

Reply:
For `database.yml.example`, you don't need to change it.
Only change username, password, database... in `database.yml` and that's all